company/okrs/2020_q1.md

```diff
@@ -0,0 +1,105 @@
+   1. VP Eng KR: Each team commits to a written work plan for each release cycle that accounts for the work estimates and the team's capacity.
+   1. VP Eng KR: Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+1. **Distribution: Resolve issues for critical customer deals and expansions**
+1. **Distribution: Improve developer productivity**
+   1. Distribution KR: Lower build times to X
+   1. Distribution KR: Automate away release captain role
+   1. Distribution KR: Infrastructure for automated deploys and releases (no human in loop required by default)
+1. **Distribution: Build features that accelerate customers through pipeline**
+   1. Distribution KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
+   1. Distribution KR: Ensure new features (LSIF, GitLab Native Integration, Automation) are accessible and easily tryable by customers once they go GA
+   1. Distribution KR: Surface information to site admins and managers to encourage expansion/upsell
+
+1. **Code intelligence: LSIF is adopted by customers**
+   1. Code intelligence KR: At least three customers are using LSIF on their primary repositories
+   1. Code intelligence KR: At least one customer is using LSIF across all of their code in languages for which there exists an LSIF indexer
+   1. Code intelligence KR: Start tracking the performance (user interaction to result) of hover tooltips, go to definition, and find references
+
+1. **Code intelligence: Increase user confidence in code intelligence provided by LSIF**
+   1. Code intelligence KR: Code intelligence UI (tooltips, references panel, etc.) indicates whether or not results are precise
+   1. Code intelligence KR: Fall back to fuzzy if the file has been modified since the last LSIF upload
+   1. Code intelligence KR: Make use of LSIF data when the current line is not part of the diff since the last LSIF upload
+
+1. **Code intelligence: Make search better using precise code intel data**
+   1. Code intelligence KR: Implement search within dependencies and/or dependents of current repo
+   1. Code intelligence KR: Implement searching for defs/refs by symbol name
+
+1. **Web: Make users aware of advanced search features**
+   1. Web KR: Start tracking time to first search and search filters per query in pings
+   1. Web KR: Shorten time to first search
+   1. Web KR: Increase average number of search filters per query
+
+1. **Web: Automation is adopted by customers**
+   1. Web KR: At least four customers are using campaigns in production
+   1. Web KR: All campaign types have been used by customers in production
+
+1. **Core Services: Search gains new capabilities which are adopted by customers**
+   1. Core Services KR: Support AND/OR/NOT operators on file contents.
+   1. Core Services KR: Write at least 2 blog posts about new search features.
```

Comment:
@rvantonder @tsenart 

Are blog posts the right medium of success here? They are inherently ephemeral. Should we prefer docs (even if they are long form), instead?

Reply:
I originally suggested this should be N pieces of technical documentation (includes usage docs, walkthrough examples, and blog posts). Likely in that order. IMO our existing search features really need some attention here (there is a lot of room for improvements on our search keywords reference and examples pages)

Reply:
Example, this exists on twitter https://twitter.com/slimsag/status/1214271545012416514 but not in our examples https://docs.sourcegraph.com/user/search/examples. We all use search, so an inclusive KR that everyone can contribute to here would be nice.

So perhaps `s/new search features/new and existing search features`

Reply:
Agree with expanding the KR beyond just structural search and also beyond just blog posts.

Reply:
```suggestion
   1. Core Services KR: Write at least 2 pieces of technical documentation (docs, walkthroughs, blog posts) about search functionality.
```